README.md

```diff
@@ -148,6 +148,7 @@ In an ideal world, your REST API follows all the [right patterns and best practi
 To use the functions that operate on collections, your API and config needs to meet some basic requirements:
 
 * The URL specified on the `fetch` config should return the full collection when the `/:id` is omitted. 
+* URLs can have optional path parameters by including `?`, such as `${process.env.HOST_URL}/users/:userId?/books`.
```

Comment:
Maybe:
> Optional path parameters can be indicated by adding a `?` suffix.

Reply:
I updated it to include this suggestion.